### PR TITLE
WSTEAMA 139 Hierarchical Promos Integration

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves WSTEAMA-139

**Overall change:**
Render the hierarchical grid if visual prominence value is HIGH. Render standard grid if visual prominence is NORMAL

**Code changes:**

- Pass `visualProminence` value from `pagaData` to `Curation`; Remove hardcoded value of `NORMAL`
- Render `HierarchicalGrid` if `visualProminence` is `HIGH`
- Render `CurationGrid` if `visualProminence` is `NORMAL`
- Rename `summaries` to `promos` to have consistency across `HierarchicalGrid` and `CurationGrid`
- Update fixture data for Storybook component 

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by rebeccamcginn** [comment]
I need to find out what box is checked for ‘visual style’ in Tipo. Currently left it as ‘none’. Easy to update once we know.

**Review by nicola-d-bbc — APPROVED** [review]

File: src/app/pages/TopicPage/Curation/index.jsx [hunk 1]
```diff
@@ -10,13 +11,17 @@ export const VISUAL_STYLE = {
...
 };
 
 // Maps a visual style and prominance to a component that renders that curation
 const components = {
   [VISUAL_STYLE.NONE]: {
     [VISUAL_PROMINANCE.NORMAL]: CurationGrid,
   },
+  [VISUAL_STYLE.NONE]: {
+    [VISUAL_PROMINANCE.HIGH]: HierarchicalGrid,
+  },
```
Comment (andrewscfc) [inline src/app/pages/TopicPage/Curation/index.jsx:23]: Can we test this mapping logic?

**Review by andrewscfc — COMMENTED** [review]
Looks great, just missing a test I think 👍

File: src/app/pages/TopicPage/Curation/index.jsx (hunk 1 above)
Comment (rebeccamcginn) [reply src/app/pages/TopicPage/Curation/index.jsx:23]: Done

**Review by pvaliani — APPROVED** [review]

**Comment by pvaliani** [comment]
Looks good! As Andrew said think it just needs some unit tests 👍

**Pull request merged**